server/.env.example

```
@@ -3,9 +3,13 @@ MONGO_DB=anomaly
 MONGO_ROOT_USERNAME=username
 MONGO_ROOT_PASSWORD=password
 CORS_ALLOWED_ORIGINS="http://localhost:1420,http://localhost:5173,http://localhost:3000,tauri://localhost,http://tauri.localhost"
+# JWT_SECRET: required, must be at least 32 characters (HS256 HMAC key).
+# In production, generate with: openssl rand -base64 48
+JWT_SECRET=dev-only-change-me-to-a-strong-random-secret-at-least-32-chars
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the MONGO_DB key

(UnorderedKey)